packages/edit-post/src/store/actions.js

```diff
@@ -509,7 +509,7 @@ export const initializeMetaBoxes = () => ( { registry, select, dispatch } ) => {
 		}
 	} );
 
-	return {
+	return dispatch( {
```

Comment:
The `return` should be removed as it doesn't do anything useful, but yeah, merely returning the action doesn't get it dispatched, like in generators. Good catch!

Reply:
Removed it 😄